chatbot/chat.py

```diff
@@ -126,7 +124,7 @@ async def on_message(room: MatrixRoom, event: RoomMessageText):
         return
 
     # construct data for command machinery
-    argv = event.body.split(" ")
+    argv = shlex.split(event.body)
```

Comment:
[shlex](https://docs.python.org/3/library/shlex.html#shlex.split) is a builtin that does exactly what we want, i.e. \*nix-like command parsing. This preserves all whitespace in quotes, while ignoring extra whitespace. For example 
```
$command     some  text       'more text'
```
would produce the array 
```
['$command', 'some', 'text', 'more text']
```